src/components/Discussion/Internal/Comment/Body.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react'
+import { css } from 'glamor'
+import { renderMdast } from 'mdast-react-render'
+
+import { DiscussionContext } from '../../DiscussionContext'
+
+import { Label } from '../../../Typography'
+import { sansSerifRegular14 } from '../../../Typography/styles'
+
+import colors from '../../../../theme/colors'
+import { mUp } from '../../../../theme/mediaQueries'
+import { useMediaQuery } from '../../../../lib/useMediaQuery'
+import { useSize } from '../../../../lib/useSize'
+
+import createCommentSchema from '../../../../templates/Comment'
+
+import { Context } from './Context'
+
+import { COLLAPSED_HEIGHT } from '../../config'
+
+const schema = createCommentSchema()
+const highlightPadding = 7
+
+const buttonStyle = {
+  outline: 'none',
+  WebkitAppearance: 'none',
+  background: 'transparent',
+  border: 'none',
+  padding: '0',
+  display: 'block',
+  cursor: 'pointer',
+  height: '100%'
+}
+
+const styles = {
+  container: css({
+    position: 'relative'
+  }),
+  highlight: css({
+    top: -highlightPadding,
+    left: -highlightPadding,
+    right: -highlightPadding,
+    bottom: -highlightPadding,
+    padding: highlightPadding,
+    width: `calc(100% + ${highlightPadding * 2}px)`,
+    backgroundColor: colors.primaryBg
+  }),
+  margin: css({
+    display: 'block',
+    marginTop: 8
+  }),
+  unpublished: css({
+    marginBottom: 8
+  }),
+  collapsedBody: css({
+    height: `${COLLAPSED_HEIGHT.mobile}px`,
+    overflow: 'hidden',
+    [mUp]: {
+      maxHeight: `${COLLAPSED_HEIGHT.desktop}px`
+    }
+  }),
+  context: css({
+    marginBottom: 10
+  }),
+  collapeToggleContainer: css({
+    borderTop: `1px solid ${colors.divider}`,
+    marginTop: 6
```

Comment:
```suggestion
    marginTop: 0
```
<img width="711" alt="Screenshot 2019-05-28 at 02 10 46" src="https://user-images.githubusercontent.com/410211/58442811-0a9ace00-80ee-11e9-9aa4-d4a6fbd7db53.png">